src/tests/Microsoft.PowerFx.Connectors.Tests/PowerPlatformConnectorTests.cs

```diff
@@ -478,5 +478,28 @@ public async Task Office365Users_SearchUsers_V2()
             Assert.IsType<StringValue>(result);
             Assert.Equal("John Doe", (result as StringValue).Value);
         }
+
+        [Fact]
+        public async Task PetStore_MultiServer()
+        {
+            using var testConnector = new LoggingTestServer(@"Swagger\PetStore.json");
+            var apiDoc = testConnector._apiDocument;
+            var config = new PowerFxConfig();
+
+            using var httpClient = new HttpClient(); // testConnector);
+
+            using var client = new PowerPlatformConnectorClient(
```

Comment:
>PowerPlatformConnectorClient [](http://example.com/codeflow?start=35&length=28)

Is PetStore a connector?
Can we access this directly as a rest API (httpClient, not PowerPlatformConnectorClient)?

Reply:
[https://petstore.swagger.io/](https://petstore.swagger.io/)